setup.py

```diff
@@ -2,47 +2,27 @@
 # // SPDX-License-Identifier: Apache-2.0
 # // Copyright : JP Morgan Chase & Co
 ###############################################################################
+# from setuptools import setup, Extension
 from setuptools import setup, Extension
 from setuptools.command.build_ext import build_ext
 import subprocess
 import os
 import sys
 
 
-environment_variable_name = "QOKIT_NO_C_ENV"
 QOKIT_PYTHON_ONLY = False
-QOKIT_NO_C_ENV = False  # used for tests only
-
-environment_variable_value = os.environ.get(environment_variable_name, None)
-
-if environment_variable_value is not None:
-    QOKIT_NO_C_ENV = True
-
 path = "./qokit/fur/c/csim/src/"
 
-sources = [os.path.join(path, "diagonal.c"), os.path.join(path, "fur.c"), os.path.join(path, "qaoa_fur.c")]
+python_only = os.environ.get("QOKIT_PYTHON_ONLY")
 
-extensions = []
-if not QOKIT_PYTHON_ONLY:
-    extensions.append(
-        Extension("simulator", sources=sources, include_dirs=[os.path.join(path, "")], extra_compile_args=["/d2FH4-"] if sys.platform == "win32" else [])
-    )
-
-
-class SimulatorBuild(build_ext):
-    def run(self):
-        try:
-            if not QOKIT_PYTHON_ONLY:
-                if QOKIT_NO_C_ENV:
-                    raise Exception("No C/C++ enviroment setup")
-                subprocess.call(["make", "-C", path])
-            super().run
-        except Exception as e:
-            print("No C/C++ enviroment setup to compile the C simulator. Installing Python Simulator")
+if python_only is not None:
```

Comment:
Can this condition be used directly on line 23 below instead of creating an intermediary `QOKIT_PYTHON_ONLY` variable?

Reply:
Need that env variable to control workflow/debug

Reply:
Looks like more tests are failing now...

Reply:
> Looks like more tests are failing now...

It should right? This is python build only no c-build

Reply:
Why did the tests start failing though? Responding in this thread, but not sure which change triggered it

Reply:
If it's intended, it should catch the error, no?